src/components/ToggleSwitch/index.tsx

```diff
@@ -39,7 +40,7 @@ const styles = StyleSheet.create({
     lineHeight: dimen.normalizeDimen(18),
     textAlign: 'center',
     textAlignVertical: 'center',
-    color: '#f4f3f4',
+    color: COLORS.whiteSmoke,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Replacing hard-coded color values with dynamic colors from a theme object improves modularity and consistency.
```diff
-    color: '#f4f3f4',
+    color: COLORS.whiteSmoke,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->